### DISCUSSION TITLE
[img2img] Not prompt guidance, but image guidance scale how to?

### DISCUSSION THREAD

**Discussion by thedyze** [hlky/stable-diffusion-webui#518] [Q&A]
Is it possible to set image guidance scale in a similar fashion as prompt guidance?

**Comment by telemole** [comment]
You might be referring to de-noising strength : 
![image](https://user-images.githubusercontent.com/22926791/188217658-908d6ea0-5fcb-4c4d-bf8b-0cf5610635e3.png)

Where a setting of 1.0 is complete destruction of original is possible, to 0, where it would keep the entire original

**Comment by thedyze** [comment]
To my understanding that's not what I'm referring to.
